web/client/components/resources/modals/Save.jsx

```diff
@@ -82,6 +84,8 @@ class SaveModal extends React.Component {
      * @return the modal for unsaved changes
     */
     render() {
+        const canEditResourcePermission = this.props.user.role === 'ADMIN' ||
```

Comment:
```suggestion
        const canEditResourcePermission = this.props.user && this.props.user.role === 'ADMIN' ||
```

Reply:
I have initialized user in default prop. @MV88

Reply:
user can be null and the default props, works only with undefined values. please update pr